pkgs/tools/misc/farge/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bash
+, bc
+, feh
+, grim
+, imagemagick
+, slurp
+, wl-clipboard
+, xcolor
+, waylandSupport ? true
+, x11Support ? true
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "farge";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "sdushantha";
+    repo = "farge";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-rTY4vQkrcyVRfIHt1qtFpxaC5bmpJ2/C6NVhEDkxef8=";
+  };
+
+  buildInputs = [ bash bc feh ]
+    ++ lib.optionals waylandSupport [ grim slurp wl-clipboard ]
+    ++ lib.optionals x11Support [ xcolor ];
+
+  installPhase = ''
```

Comment:
Missing phase hooks: https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/missing-phase-hooks.md

Reply:
Added them

Reply:
Don't forget to squash your commits, there should only be 2 of them.

Reply:
Yep, done